contorollers/feed.js

```diff
@@ -53,7 +53,7 @@ exports.getFeed = async (req, res, next) => {
 exports.addLikeUser = async (req, res, next) => {
   try {
     const { id } = req.params;
-    const userId = req._id;
+    const userId = req.body.userId;
```

Comment:
```suggestion
    const { userId } = req.body;
```
위와 동일하게 해줘도 좋을 것 같습니다.👍
고생하셨습니다.

Reply:
넵 스타일 통일하겠습니다.